openshift/release/generate-release.sh

```diff
@@ -19,10 +19,6 @@ readonly SERVING_POST_INSTALL_JOBS_YAML=${YAML_OUTPUT_DIR}/serving-post-install-
 if [[ "$VERSION" == "ci" ]]; then
   # Do not use devel as operator checks the version.
   VERSION="v1.2.0"
-else
```

Comment:
Was this never used?

Reply:
`create-release-branch.sh` has an issue and so fixed it and added this condition again.